setup.py

```diff
@@ -0,0 +1,68 @@
+import os
```

Comment:
You need a `setup.py` for development and installation no? And tox is for running the tests, regardless if it's an app or a lib